### ISSUE TITLE
[BUG] Masking validation not accepting correct values

### ISSUE THREAD

**Issue by davidgraat** [formio/formio.js#2387]
## Environment
Please provide as many details as you can:

- Hosting type
  - [x] Form.io
  - [ ] Local deployment
    - Version: 
- Formio.js version: v4.9.0-beta.2
- Frontend framework: Javascript
- Browser: Chrome
- Browser version: 79.0.3945.130
- Form snippet code using https://unpkg.com/formiojs@latest/dist/formio.full.min.js, which currently resolves to https://unpkg.com/formiojs@4.9.0-beta.2/dist/formio.full.min.js.

## Steps to Reproduce
1. Create a form on portal.form.io
2. Add a Phone Number field.
3. Use the embed code to add the form to an html page.
4. Open the page in your browser and fill in the phone number field with a phone number of the proper format, such as "(555) 555-5555".

## Expected behavior
The form should accept the entry with no validation error.

## Observed behavior
Any value I type is treated as invalid, with the message "Phone Number does not match the mask."
Generalizing this issue a bit more, it seems that the input masking is not allowing any values.  I believe this behavior was introduced yesterday.  If I switch to version 4.9.0-beta.1 of the script, then I can get past this error for the time being.

## Example
![image](https://user-images.githubusercontent.com/47161159/74057032-ae8c8700-49a8-11ea-8a4a-60dbd2375bb4.png)
```
<html>
  <head>
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
    <link rel="stylesheet" href="https://unpkg.com/formiojs@latest/dist/formio.full.min.css">
    
    <!--Forcing use of formio.js 4.9.0-beta.2 that is causing the issue.-->
    <!--<script src="https://unpkg.com/formiojs@latest/dist/formio.full.min.js"></script>-->
    <script src="https://unpkg.com/formiojs@4.9.0-beta.2/dist/formio.full.min.js"></script>
  </head>
  <body>
    <div id="formio"></div>
    <script type="text/javascript">
      window.onload = function() {
        Formio.createForm(document.getElementById('formio'), {
            "components": [ {
              "autofocus": false,
              "input": true,
              "tableView": true,
              "inputType": "tel",
              "inputMask": "(999) 999-9999",
              "label": "Phone Number",
              "key": "phoneNumber",
              "placeholder": "",
              "prefix": "",
              "suffix": "",
              "multiple": false,
              "protected": false,
              "unique": false,
              "persistent": true,
              "hidden": false,
              "defaultValue": "",
              "clearOnHide": true,
              "validate": {
                "required": false
              },
              "type": "phoneNumber",
              "labelPosition": "top",
              "inputFormat": "plain",
              "tags": [],
              "conditional": {
                "show": "",
                "when": null,
                "eq": ""
              },
              "properties": {}
            },
            {
              "autofocus": false,
              "input": true,
              "label": "Submit",
              "tableView": false,
              "key": "submit",
              "size": "md",
              "leftIcon": "",
              "rightIcon": "",
              "block": false,
              "action": "submit",
              "disableOnInvalid": false,
              "theme": "primary",
              "type": "button"
            }
          ]
        });
      };
    </script>
  </body>
</html>
```

**Comment by wag110894** [comment]
Hello @davidgraat, thank you for reaching out!

We are able to replicate this on our side. We have created a ticket and have added this to our backlog. We welcome a pull request or have an option to expedite this issue. Please contact support@form.io is you are in expediting this.

Have a great day!

**Comment by jhen1422** [comment]
@davidgraat We apologize about the confusion. This behavior is actually expected. Input mask will validate until the field has been completed. Once the input mask has been honored the validation will be removed. You can change the validation type to 'Blur' so the validation will occur after field loses focus.

**Comment by davidgraat** [comment]
@jhen1422 @wag110894 If you look at my example, you will see the phone number "(555) 555-5555" does satisfy the mask.  However, the validation flags it as an error.  How is this expected behavior?

**Comment by jhen1422** [comment]
@davidgraat Does this problem persists using a previous formio.js version?

**Comment by davidgraat** [comment]
@jhen1422 Looks like this is fixed in formiojs@4.9.0-beta.3.  Sorry, the timing was just bad... It broke last week Thursday as I was preparing for a demo for my leadership since we are considering your Enterprise product.  I'm guessing it was fixed in this commit: https://github.com/formio/formio.js/commit/92a2f2d06882b4718389e8d51ea3da6b16d52c84

Let me follow up with this:  A form that I create on https://portal.form.io is giving me embed code that references https://unpkg.com/formiojs@latest/dist/formio.full.min.js.  Today this resolves to https://unpkg.com/formiojs@4.9.0-beta.3/dist/formio.full.min.js.  Last week Thursday/Friday it resolved to https://unpkg.com/formiojs@4.9.0-beta.2/dist/formio.full.min.js, which caused my forms to be broken for my demo.  (At the time I worked around it by changing the reference to 4.9.0-beta.1.)  Do you expect that the embed code is referencing a script that is marked as "beta"?
